tests/bulk-tx/src/bulk_tx.ts

```diff
@@ -0,0 +1,144 @@
+import { ApiPromise, WsProvider, Keyring } from '@polkadot/api';
+import { SignerResult } from '@polkadot/api/types';
+import { KeyringPair } from '@polkadot/keyring/types';
+import type { EventRecord, ExtrinsicStatus, H256, SignedBlock } from '@polkadot/types/interfaces';
+import type { ISubmittableResult, SignatureOptions } from '@polkadot/types/types';
+
+const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+const keyring = new Keyring({ type: 'sr25519' });
+
+//batch size and payload size are set here
+const batch:number = 3;
+const size:number = 100;
```

Comment:
Not sure in what unit the size is defined. bits, bytes, KB, MB, .. ?

Reply:
bytes. So if its 100, then its 100bytes